src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -134,6 +134,7 @@
          Platform="$(PlatformTarget)"
          Prefer32Bit="$(Prefer32Bit)"
          PreferredUILang="$(PreferredUILang)"
+         ProjectName="$(MSBuildProjectName)"
```

Comment:
Do we still have copies of this task floating around in other repos? I thought WPF had one but I couldn't find it.

Reply:
I _think_ we got rid of the other places where `Csc` task was invoked directly. There have been a few other cases where we added properties recently and haven't had much issue. If we find some though can update them.

Reply:
I was wrong https://github.com/dotnet/msbuild/pull/9786